clean_data.py

```diff
@@ -188,21 +181,25 @@ def main(argv):
       for line in fp:
         line = line.strip()
         print(create_hmac(line, FLAGS.hmac_key, hash_type=hashlib.sha3_384))
-  return
 
   df = pd.read_csv(FLAGS.input,
                    on_bad_lines='warn',
-                   na_values=['NR', '\n',
+                   na_values=['\n',
                               '? * Line 1, Column 1\n  Syntax error: value, '
                               'object or array expected.\n* Line 1, Column 1\n'
                               '  A valid JSON document must be either an array '
                               'or an object value.']) 
+  #Replace NR values with a million to ensure it is not included in calculations of PTA, and to caution the user regarding no responses values at a particular frequency
```

Comment:
This comment needs to be up top too.   This file does nothing but prepare the clinic data, so list the assumptions and final output statement up top, not buried in the code.